misc/pytest/lib/query_suites.py

```diff
@@ -0,0 +1,30 @@
+
+import os
+import pytest
+
+@pytest.fixture
+def check_query_suite(codeql, cwd, expected_files, semmle_code_dir):
+    def ret(query_suite):
+        actual = codeql.resolve.queries(query_suite, _capture=True).strip()
```

Comment:
`strip` is already done by the capture
```suggestion
        actual = codeql.resolve.queries(query_suite, _capture=True)